docs/source/snakemake/debugging.md

```diff
@@ -0,0 +1,47 @@
+# Debugging Snakemake
+
+## Local Development
+
+When debugging a Snakemake workflow, it's helpful to run the JIT step locally instead of re-registering your workflow everytime you want to test a change. To address this, the Latch SDK supports local development for Snakemake workflows.
+
+If you are not familiar with the `latch develop` command, please read about [Local Development](../basics/local_development.md) before continuing.
+
+---
+
+Run the following command to use `latch develop` with your Snakemake workflow:
+
+```console
+$ latch develop . --snakemake
+```
+
+Once you are in the container, run the following command to run the JIT task in "dry run" mode.
+
+```console
+$ export LATCH_SNAKEMAKE_DRY_RUN=true
+```
```

Comment:
we should set this automatically in develop sessions, shouldn't be left to the user

Reply:
https://github.com/latchbio/centromere-old/pull/1